vault/logical_system.go

```diff
@@ -4008,6 +4008,8 @@ func (b *SystemBackend) pathInternalOpenAPI(ctx context.Context, req *logical.Re
 	// be received from plugin backends.
 	doc := framework.NewOASDocument()
 
+	genericMountPaths := d.Get("generic_mount_paths").(bool)
```

Comment:
```suggestion
	genericMountPaths, _ := d.Get("generic_mount_paths").(bool)
```

One more to avoid panic :)

Reply:
I think this particular `Get` guards against this if the field doesn't exist and uses the type default instead. But I'm happy to add it for security sake. haha

Reply:
I think what we want to guard againt is the type being a non-bool (e.g. a string) which could cause a panic.

Reply:
okay. I'm confused. If it is a string, doesn't acknowledging the `Ok` make it default to false? thus avoiding the panic? Maybe I misunderstood the discussion.

Also, b/c the field data sets this `generic_mount_paths` as a boolean, it will come back as a boolean w/ this particular `Get` as it decodes the values into the proper types. There's no way for it to come through as a string.

Reply:
oooh. wait. sorry. I misunderstood. I did add the suggested fix